WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTrackingTableViewCell.swift

```diff
@@ -0,0 +1,102 @@
+import UIKit
+
+/// Displays shipment tracking information associated to an Order.
+///
+final class OrderTrackingTableViewCell: UITableViewCell {
+    @IBOutlet private var topLine: UILabel!
+    @IBOutlet private var bottomLine: UILabel!
+
+    @IBOutlet private var actionButton: UIButton!
+
+    var onActionTouchUp: (() -> Void)?
+
+    var topText: String? {
+        get {
+            return topLine.text
+        }
+        set {
+            topLine.text = newValue
+            configureTopLineForVoiceOver()
+        }
+    }
+
+    var bottomText: String? {
+        get {
+            return bottomLine.text
+        }
+        set {
+            bottomLine.text = newValue
+            configureBottonLineForVoiceOver()
+        }
+    }
+
+    var actionButtonNormalText: String? {
+        get {
+            return actionButton.title(for: .normal)
+        }
+        set {
+            actionButton.setTitle(newValue, for: .normal)
+            configureActionButtonForVoiceOver()
+        }
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureTopLine()
+        configureBottomLine()
+        configureActionButton()
+    }
+
+    private func configureTopLine() {
+        topLine.applyHeadlineStyle()
+    }
+
+    private func configureBottomLine() {
+        bottomLine.applySubheadlineStyle()
+    }
+
+    private func configureActionButton() {
+        actionButton.applySecondaryButtonStyle()
+        configureActionButtonForVoiceOver()
+    }
+}
+
+extension OrderTrackingTableViewCell {
+    @IBAction func actionButtonPressed(_ sender: Any) {
+        onActionTouchUp?()
+    }
+}
+
+/// Accessibility
+///
```

Comment:
Super tiny style suggestion → for `extension`s defined in the same swift file, we've been _trying_ have 2 vertical whitespace lines after the closing brace with a `MARK: -` comment immediately above the `extension XXXXXXX {` line. e.g.: 

```
    ...
    //end of some extension or class or struct
}


// MARK: - Accessibility
//
private extension XXXXXX {
```

feel free to ignore if you want. 😄